explorer/client/.gitignore

```diff
@@ -18,3 +18,5 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+cypress/screenshots
```

Comment:
One of the main reasons we discussed on Slack for using Cypress for testing was that it would automatically generate videos that our design team could then review. This saves the engineers time in manually creating screenshots and videos such that the designers can see what is going on. Hence, perhaps the photos and videos generated by Cypress should not be gitignored? Furthermore, could we have these videos/screenshots generated for both a Desktop and Mobile view?

Reply:
These aren't quite what you expect. These are debugging screenshots / videos that get generated on failed runs. We do want visual regression testing, and cypress does unlock that, but we'll do that through some other mechanism rather than having users check-in assets (which inflates git repo size, and is a poor DX).

What we'll probably do is generate screenshots in CI during runs, and then store them somewhere like S3.

Reply:
The videos are generated on successful as well as unsuccessful runs - see the file `explorer/client/cypress/videos/e2e.cy.ts` - which is generated after each test run, and see documentation here: https://docs.cypress.io/guides/guides/screenshots-and-videos#Control-which-videos-to-keep-and-upload-to-Dashboard. Cypress compresses the videos which reduces their memory impact. This is why I can upload the video below:

https://user-images.githubusercontent.com/11377188/183870938-713fe64e-7c3e-4670-9c6e-b93fc20d26cd.mp4

Reply:
In the past, @mystie711 has taken the screenshots that we have generated and then edited these in Figma to explain required changes to the website. Furthermore, @Clay-Mysten is keen that we create Documentation for the Sui Explorer website. Based on this, it would be useful to have regularly updated screenshots of key parts of the website that could be incorporated in this documentation: Home Page, Transaction Results Page, Object Results Page and Address Results Page. 

Could you insert `cy.screenshot` throughout the tests to generate these images? These images would be automatically placed in `explorer/client/cypress/screenshots`. 

As a prelude to this, it would be useful to add an action that accepts the cookies in order to prevent this reminder from obscuring part of the web page.

Reply:
The screenshots in the Docs Portal are very old - see https://docs.sui.io/explore/panzerdogs
![Panzer Dogs Image](https://docs.sui.io/suipanzerdogs5.png)

Reply:
> In the past, @mystie711 has taken the screenshots that we have generated and then edited these in Figma to explain required changes to the website. Furthermore, @Clay-Mysten is keen that we create Documentation for the Sui Explorer website. Based on this, it would be useful to have regularly updated screenshots of key parts of the website that could be incorporated in this documentation: Home Page, Transaction Results Page, Object Results Page and Address Results Page.
> 
> Could you insert `cy.screenshot` throughout the tests to generate these images? These images would be automatically placed in `explorer/client/cypress/screenshots`.
> 
> As a prelude to this, it would be useful to add an action that accepts the cookies in order to prevent this reminder from obscuring part of the web page.

Issue filed for the docs here:
https://github.com/MystenLabs/sui/issues/3899

Reply:
The use-case for documentation specifically is interesting and one we could look into.

Understood that videos and images can be generated on successful and unsuccessful runs. I still don't think tracking most of them in Git is the right call. If anything, we should be taking the runs in CI and uploading those to an S3 bucket for this purpose.

There are some longer-term challenges to this as well. My long-term goal is to remove the static server, and move everything to be a localnet setup with data populated through a script. This would churn the object ID's on every run, which isn't ideal for screenshotting purposes. Once the app is more componentized this is a little easier to solve through component-level mocking, but it's still going to be an interesting problem to try to solve.

Reply:
We'll handle the generation of screenshots in the following separate issue https://github.com/MystenLabs/sui/issues/4047, and shift discussion here. Looking forward to seeing this in action.